peripherals/wwdg/wwdg.yaml

```diff
@@ -0,0 +1,38 @@
+# The most common features, found on Basic timers TIM6 and TIM7,
+# and found on all more advanced timers.
+
+WWDG:
+  CR:
+    _modify:
+      WDGA: 
+        description: "Activation bit for the watchdog. This bit is set by software and only cleared after a reset."
+        bitOffset: 7
+        bitWidth: 1
+      T:
+        description: "These bits contain the value of the watchdog counter. A reset is produced when it rolls over from 0x40 to 0x3F (T6 us cleared)."
+        bitOffset: 0
+        bitWidth: 7
```

Comment:
It seems like the WDGA and T fields are already present in the SVD, so they don't need modifying (the SVD already has the description, bitOffset, and bitWidth). Or is it broken/missing in some SVD?

The thing to do instead is give either enumerated values (perhaps Disabled and Enabled for WDGA), or a range of permissible values (such as [0, 127] for T). For example:

```
CR:
  WDGA:
    Disabled: [0, "Watchdog disabled"]
    Enabled: [1, "Watchdog enabled"]
  T: [0, 127]
```